buildkite/graphql/team.graphql

```diff
@@ -0,0 +1,84 @@
+fragment TeamFields on Team {
+	id
+	uuid
+	name
+	# @genclient(pointer: true)
+	description
+	slug
+	privacy
+	isDefaultTeam
+	defaultMemberRole
+	membersCanCreatePipelines
+}
+
+query getTeam($id:ID!) {
```

Comment:
Could this  query be moved into the `getNode` query in `node.graphql`?

Reply:
That feels a bit refactor-y? It would require a change to most of the source files in this.

Reply:
Might be a little bit - I _think_ the only difference would be the type naming of the generated code, more keeping things in one place thing

Reply:
@james2791 if it's an agreed upon _thing_ then I can change it; it felt experimental at the time. Feels like maybe something else we migrate to but I'm not sure if it's a requirement?

@jradtilbrook can decide.

Reply:
Theres a few things on the refactor list already, lets lump it on there. I _think_ the single `getNode` is the way to go

Reply:
As long as it works in the end 👍